packages/kotti-ui/source/kotti-modal/KtModal.vue

```diff
@@ -29,6 +31,12 @@ export default defineComponent<KottiModal.PropsInternal>({
 	name: 'KtModal',
 	props: makeProps(KottiModal.propsSchema),
 	setup(props, { emit, slots }) {
+		const wrapperRef = ref<HTMLElement | null>(null)
+
+		onMounted(() => {
+			if (wrapperRef.value) window.document.body.appendChild(wrapperRef.value)
```

Comment:
I don't think this is legal to just do. What happens when content updates and Vue needs to re-render something? Is there any place in the Vue documentation that encourages this? How did you find out about this in the first place?

I don't know how it's broken, but this is something that you usually shouldn't be able to do with a VDOM

Reply:
Document shouldn’t change with navigation and memory leaks are removed after a refresh.

Also, please keep in mind that this is just an example of what’s wrong with manipulating VDOM with native DOM operations. Anything could be wrong with this, it’s just the first thing I found.

Reply:
@FlorianWendelborn @carsoli about the memory leak, I missed a removeChild on Unmount. I fixed that and tested locally and with Yoda. Although it seems to work even with data being updated inside the modal, I agree that manipulating the VDom is not a good approach. 
In any case, the PR is now updated with an implementation using Tippy.